File: hello-world-python/handler.py
import json
import time

def hello(event, context):
    body = {
        "message": "Go Serverless v1.0! Your function executed successfully again and again!",
        "input": event
    }

    response = {
        "statusCode": 200,
        "body": json.dumps(body)
    }

    time.sleep(4)

    return response

    # Use this code if you don't use the http event with the LAMBDA-PROXY
    # integration
    """
    return {
        "message": "Go Serverless v1.0! Your function executed successfully!",
        "event": event
    }
    """

File: mailman/services/users/handler.js
'use strict';

module.exports.register = async event => {
  
  //console.log('Registering user');

  const userInfo = {
    name: "John",
    email: "john@example.com"
  };

    return {
      statusCode: 200,
      body: JSON.stringify(
        {
          message: 'User Registered',
          input: event,
        },
        null,
        2
      ),
    };

  // Use this code if you don't use the http event with the LAMBDA-PROXY integration
  // return { message: 'Go Serverless v1.0! Your function executed successfully!', event };
};
